exercises/concept/rpn-calculator-inspection/.docs/introduction.md

```diff
@@ -0,0 +1,3 @@
+TODO: copy-paste links introduction
+
+TODO: copy-paste task introduction
```

Comment:
I wonder if there will need to be some more code examples for students to be able to solve this.

Reply:
I initially wrote introductions for both concepts with code examples. Then I copied that to `about.md`s and removed the code examples from the introductions because I remembered that Jeremy suggested multiple times to keep the code examples in introductions to minimum, only new syntax allowed. There's no new syntax here in this exercise 🤷. I agree that it will be challenging to figure out this exercise without code examples...